lua/curry/theme.lua

```diff
@@ -99,32 +99,100 @@ local highlight_groups = {
 	-- To get a list of all possible groups
 	-- `rg --no-filename -o -g '**/highlights.scm' '^\s*?[^;].*? (@[^\s_)"]+?)[\s)]*$' -r '$1' | sort | uniq`
 
-	["@keyword.gitcommit"] = { link = "Function" },
-	["@field"] = { fg = c.cyan, bg = "NONE" },
-	["@parameter"] = { fg = c.red, bg = "NONE" },
-	["@property"] = { fg = c.cyan, bg = "NONE" },
-	["@type.qualifier"] = { fg = c.orange, bg = "NONE" },
-	["@type.qualifier.rust"] = { fg = c.orange, bg = "NONE", bold = true },
+	-- Identifiers
+	["@variable"] = { link = "Identifier" },
+	["@variable.builtin"] = { fg = c.red },
+	["@variable.parameter.builtin"] = { fg = c.purple },
+	["@variable.parameter"] = { fg = c.red, bg = "NONE" },
+	["@variable.member"] = { fg = c.cyan, bg = "NONE" },
 
-	-- Treesitter markdown
-	["@text.title.1.marker.markdown"] = { fg = c.cyan, bg = "NONE" },
-	["@text.title.2.marker.markdown"] = { fg = c.cyan, bg = "NONE" },
-	["@text.title.3.marker.markdown"] = { fg = c.cyan, bg = "NONE" },
-	["@text.title.4.marker.markdown"] = { fg = c.cyan, bg = "NONE" },
-	["@text.title.5.marker.markdown"] = { fg = c.cyan, bg = "NONE" },
-	["@text.title.6.marker.markdown"] = { fg = c.cyan, bg = "NONE" },
-	["@text.title.1.markdown"] = { fg = c.blue },
-	["@text.title.2.markdown"] = { fg = c.blue },
-	["@text.title.3.markdown"] = { fg = c.blue },
-	["@text.title.4.markdown"] = { fg = c.blue },
-	["@text.title.5.markdown"] = { fg = c.blue },
-	["@text.title.6.markdown"] = { fg = c.blue },
+	["@constant"] = { link = "Constant" },
+	["@constant.builtin"] = { fg = c.purple },
+	["@constant.macro"] = { link = "Macro" },
 
-	["@text.literal.block.markdown"] = { fg = c.fg },
-	["@text.literal.markdown_inline"] = { fg = c.purple },
+	["@label"] = { link = "Label" },
+	["@module"] = { link = "Structure" },
+	["@module.builtin"] = { fg = c.brown },
 
-	["@punctuation.special.markdown"] = { fg = c.brown },
+	-- Literals
+	["@string"] = { link = "String" },
 
```

Comment:
Are the empty line here intentional?

Reply:
Yeah I was trying to separate `@string` from the rest of `@string` related highlight groups, which are overrides

Reply:
Okay, seems good to me!